website/docs/tutorials/08-cairo-1-support/03-cheatcodes/deploy_contract.md

```diff
@@ -0,0 +1,33 @@
+# `deploy_contract`
+
+```cairo
+fn deploy_contract(contract: felt252, calldata: Array::<felt252>) -> 
+Result::<felt252, RevertedTransaction>
+```
+
+Declares and deploys a contract given its name defined in the [protostar.toml](../../04-configuration-file.md) configuration
+file.
+
+```cairo title="Example"
+use result::ResultTrait;
+use array::ArrayTrait;
+
+fn my_function() {
+
+    let mut calldata = ArrayTrait::new();
+    calldata.append(10);
+    calldata.append(11);
+    calldata.append(12);
+
+    let address = deploy_contract('mycontract', calldata).unwrap();
+
+    // ...
+}
+```
+
+Errors raised by the constructor can be handled in the same way as for [deploy](deploy.md)
+
+
+:::info
+`deploy_contract` is just a syntactic sugar over executing cheatcodes `declare` -> `prepare` -> `deploy` separately, and it's what it does under the hood.
```

Comment:
```suggestion
`deploy_contract` is just syntactic sugar over executing cheatcodes `declare` -> `prepare` -> `deploy` separately, and that's what it does under the hood.
```

Reply:
Same suggestion as for the other file here ⬆️

Reply:
https://github.com/software-mansion/protostar/pull/1840/commits/8534705c8eb65ab65d4d5399e8e600b7388f3316